tests/invert_test.cpp

```diff
@@ -324,20 +371,26 @@ int main(int argc, char **argv)
   if (inv_multigrid) destroyMultigridQuda(mg_preconditioner);
 
   // Compute performance statistics
-  if (Nsrc > 1) performanceStats(time, gflops, iter);
+  if (Nsrc > 1 && !use_split_grid) performanceStats(time, gflops, iter);
 
   // Perform host side verification of inversion if requested
   if (verify_results) {
-    verifyInversion(out->V(), (void **)outMulti, in->V(), check->V(), gauge_param, inv_param, gauge, clover, clover_inv);
+    for (int i = 0; i < Nsrc; i++) {
+      verifyInversion(out[i]->V(), _hp_multi_x[i].data(), in[i]->V(), check->V(), gauge_param, inv_param, gauge, clover,
+                      clover_inv);
+    }
   }
 
   // Clean up memory allocations
-  delete in;
-  delete out;
   delete check;
-  free(outMulti);
   if (multishift > 1) {
-    for (int i = 0; i < multishift; i++) delete qudaOutMulti[i];
+    for (auto p : out_multishift) { delete p; }
+  }
+
+  for (auto p : in) { delete p; }
+  for (auto p : out) { delete p; }
+  for (auto p : out_multishift) {
```

Comment:
this is a duplicate from a few lines above.

Reply:
Thanks for the catch.